Neos.ContentRepository.Core/Classes/CommandHandler/CommandBus.php

```diff
@@ -29,7 +30,10 @@ public function __construct(
     }
 
     /**
-     * @return EventsToPublish|\Generator<int, EventsToPublish>
+     * The handler only calculate which events they want to have published,
+     * but do not do the publishing themselves
+     *
+     * @return EventsToPublish|YieldedEventsToPublish
```

Comment:
Is that really easier to read than
```suggestion
     * @return EventsToPublish|\Generator<EventsToPublish>
```
?

Reply:
adjusted :) (without complaining because i did all the argumentation myself in my head ... over dayyyyys :D)